apps/web/pages/api/auth/[...nextauth].tsx

```diff
@@ -164,6 +164,7 @@ if (IS_GOOGLE_LOGIN_ENABLED) {
     GoogleProvider({
       clientId: GOOGLE_CLIENT_ID,
       clientSecret: GOOGLE_CLIENT_SECRET,
+      allowDangerousEmailAccountLinking: !hostedCal,
```

Comment:
```suggestion
      allowDangerousEmailAccountLinking: true,
```

We can trust Google to work with verified email addresses 🙏

Reply:
i think that should be fine @deepakprabhakara no?

Reply:
Yes, this should be fine but let me just double check.

Reply:
This works well and is only limited to the provider, so we can enable it for SAML as well. I'll push through the new changes